third_party/ibis/ibis_addon/operations.py

```diff
@@ -618,13 +619,21 @@ def execute_epoch_seconds_new(op, data, **kwargs):
         return epoch_series
 
 
+def _list_tables(self, like=None, database=None) -> list:
+    """Override of BaseAlchemyBackend.list_tables that does not include views in the result."""
```

Comment:
```suggestion
    """Override of BaseAlchemyBackend.list_tables that does not include views in the result. See original method at https://github.com/ibis-project/ibis/blob/5.1.0/ibis/backends/base/sql/alchemy/__init__.py#L101"""
```
Something like that, I think it'll be useful to have the source handy as I went to look for it to understand how it looked like before